<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pyrrhic REE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="gbReg">
      <property name="title">
       <string>Regular Expression Pattern</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="tedReg"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbFlags">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="title">
       <string>Flags</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="chkCase">
         <property name="toolTip">
          <string>Perform case-insensitive matching</string>
         </property>
         <property name="text">
          <string>Ignore Case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkMulti">
         <property name="toolTip">
          <string>^ matches at the beginning of the string and at the beginning of each line within the string, immediately following each newline. Similarly, the $ metacharacter matches either at the end of the string and at the end of each line (immediately preceding each newline).</string>
         </property>
         <property name="text">
          <string>Multi Line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkDot">
         <property name="toolTip">
          <string>Makes the '.' special character match any character at all, including a newline; without this flag, '.' will match anything except a newline.</string>
         </property>
         <property name="text">
          <string>Dot All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkVerbose">
         <property name="toolTip">
          <string>This flag allows you to write regular expressions that are more readable by granting you more flexibility in how you can format them. When this flag has been specified, whitespace within the RE string is ignored, except when the whitespace is in a character class or preceded by an unescaped backslash; this lets you organize and indent the RE more clearly. This flag also lets you put comments within a RE that will be ignored by the engine; comments are marked by a '#' thatâ€™s neither in a character class or preceded by an unescaped backslash.</string>
         </property>
         <property name="text">
          <string>Verbose</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkLocale">
         <property name="toolTip">
          <string>Make w, W, b, and B, dependent on the current locale.</string>
         </property>
         <property name="text">
          <string>Locale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkAscii">
         <property name="toolTip">
          <string>Make w, W, b, B, s and S perform ASCII-only matching instead of full Unicode matching. This is only meaningful for Unicode patterns, and is ignored for byte patterns.</string>
         </property>
         <property name="text">
          <string>Ascii</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbString">
      <property name="title">
       <string>Search String</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="tedString"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabResults">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabMatch">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTextBrowser" name="tebMatch"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMatchAll">
       <attribute name="title">
        <string>Find All</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTextBrowser" name="tebMatchAll"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGroup">
       <attribute name="title">
        <string>Group</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTextBrowser" name="tebGroup"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionImport_URL"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionRegex_Reference_Guide"/>
    <addaction name="actionRegex_Library"/>
    <addaction name="actionPython_Regex_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegex_Reference_Guide">
   <property name="text">
    <string>Regex Reference Guide</string>
   </property>
  </action>
  <action name="actionRegex_Library">
   <property name="text">
    <string>Regex Library</string>
   </property>
  </action>
  <action name="actionPython_Regex_Help">
   <property name="text">
    <string>Python Regex Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionShow_Variables">
   <property name="text">
    <string>Show Variables</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionImport_URL">
   <property name="text">
    <string>Import URL</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>regex_changed()</slot>
  <slot>string_changed()</slot>
 </slots>
</ui>
